tests/compilation_tests/robot_test/execute_setup_test.py

```diff
@@ -72,5 +70,16 @@ def test_radio_session_setup(self):
         robot_radio_session = RadioModule(None)
         self.assertEqual(robot_radio_session.connected, False)
 
+    #test motor_controller_setup calls the motor controller setup function, which will 
+    #print a series of commands. Make sure they are in the order of: 
+    #'go_forward', 'turn_left', 'turn_right', 'reverse', 'stop'
```

Comment:
is this a manual check? the assertEqual is not checking for this

Reply:
Not sure (as I didn't write this just copy-pasted) -- to preface where this change came from, basically I noticed there were two copies of execute_setup_test when I went through all the test files to see where we were initializing Robot. I combined them since it didn't make sense to have two duplicate files, and added any lines that differed between them!

Again, separate task from the purpose of this PR, but a problem nonetheless. Added to list of ongoing new tasks that you have pointed out previously in the PR summary